File: Easy/AddBinary/SolutionTest.java
package Easy.AddBinary;

import org.junit.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class SolutionTest{

    @Test
    public void addBinaryTest1(){
        assertEquals("100", Solution.addBinary("11", "1"));
    }

    @Test
    public void addBinaryTest2(){
        assertEquals("10101", Solution.addBinary("1010", "1011"));
    }
}

File: Easy/AddBinary/SolutionTest.py
from unittest import TestCase, main
from Solution import Solution

class SolutionTest(TestCase):
    def test1_addBinary(self):
        self.assertEqual("100", Solution.addBinary("11", "1"))


    def test2_addBinary(self):
        self.assertEqual("10101", Solution.addBinary("1010", "1011"))


if __name__ == "__main__":
    main()
